Test/client_test.py

```diff
@@ -41,17 +40,22 @@ def test_index_page2(client):
     response = client.get('/')
     assert response.status_code == 302
 
-def test_index_page_with_mock(mock_get_requests, client):
+@patch('pymongo.MongoClient')
+def test_index_page_with_mock(mock_mongo_card, client):
+    mock_mongo_card.return_value["ToDo"]["All Items"].find = substitute_mongo_api_mock
     response = client.get('/items/get_all_cards')
     assert response.status_code == 200 
-    
-def test_index_page_with_mock2(mock_get_requests, client):
-    response = client.get('/items/get_all_cards')
-    data = Trello.get_all_cards_from_board(response)
-    data2 = json.loads(data)
-    assert "Test to do item" in data
-    assert "12345" in data
-    assert "67890" in data
-    assert "2020-08-01T12:52:06.278Z" in data
-    assert data2[0] == {'id': '12345', 'name': 'Test to do item', 'idList': '67890', 'dateLastActivity': '2020-08-01T12:52:06.278Z'}
-    assert data2[0]['name'] == "Test to do item"
\ No newline at end of file
+
+
+# Cant get below to work!! Get AssertionError assert <MagicMcok.... == {'_id':'1234...} HELP
+#@patch('pymongo.MongoClient')
+#def test_index_page_with_mock2(mock_mongo_card, client):
```

Comment:
It's confusing we're patching the client but renaming it as a mock card - this might be partly what is leading to confusion with what's being tested